resources/scss/_about-settings.scss

```diff
@@ -0,0 +1,53 @@
+$font-stack: "Rubik", sans-serif;
```

Comment:
@ZadenRB we can define these vars in a file called `vars.scss` and import that in `site.scss` if we do that they should be available everywhere we need them. That way we dont have to repeat them in every scss module.